### PR TITLE
script to delete AWS backup recovery points

### PR DESCRIPTION
**Pull request by haicasgox**
script to delete AWS backup recovery points

### CODE REVIEW

File: data/blog/aws/tự động hoá xoá recovery points.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+<center> Thay đổi giá trị lên thời gian timeout đã request increase thành công </center>
+
+![Service Quotas](/static/images/assets/api_gateway_integration_timeout_successful_update.png)
+
+<center> Giá trị integration timeout sau cập nhật </center>
+
+## Kết luận:
+
+Bài toán integration timeout đã có thuốc giải. Mong rằng trong tương lai gần, AWS sẽ hỗ trợ thêm các loại API Gateway còn lại.
```
Comment (hungran) [inline data/blog/aws/tự động hoá xoá recovery points.mdx]: wrong hole?

Comment (hungran) [reply data/blog/aws/tự động hoá xoá recovery points.mdx]: @haicasgox

Comment (haicasgox) [reply data/blog/aws/tự động hoá xoá recovery points.mdx]: Oops!!! Thanks for your comment. The content has been re-uploaded.

**Review by hungran — APPROVED** [review]

**Review by qmauvnt — APPROVED** [review]
LGTM!

**Pull request merged**